src/Microsoft.Health.Dicom.Blob/Extensions/AzureStorageErrorExtensions.cs

```diff
@@ -46,6 +46,8 @@ public static bool IsConnectedStoreCustomerError(this RequestFailedException rfe
     public static bool IsStorageAccountUnknownHostError(this Exception exception)
     {
         return exception.Message.Contains("No such host is known", StringComparison.OrdinalIgnoreCase) ||
-            (exception is AggregateException ag && ag.InnerExceptions.All(e => e.Message.Contains("No such host is known", StringComparison.OrdinalIgnoreCase)));
+            exception.Message.Contains("Name or service not known", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
This was added because of Karthik's instance that was failing for this error https://microsofthealth.visualstudio.com/Health/_workitems/edit/114214

Reply:
Moved this file to Core so it can be shared with DataStoreRequestFailedException